components/gauges/circular/overview.md

```diff
@@ -12,30 +12,15 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/circular-gauge" target="_blank">Telerik Circular Gauge for Blazor</a> represents [numerical values]({%slug circular-gauge-pointers%}) on a circular [scale]({%slug circular-gauge-scale%}).
 
-#### This article is separated in the following sections: 
-
-* [Basics](#basics)
-
-* [Features](#features)
-
-* [Methods](#methods)
-
-## Basics
-
->caption To add a Telerik Circular Gauge for Blazor to your application:
+## Creating Blazor Circular Gauge
 
 1. Add the `<TelerikCircularGauge>` tag.
-
 1. Add one or more instance of the `<CircularGaugePointer>` to the `<CircularGaugePointers>` collection.
```

Comment:
It doesn't seem like we support multiple pointers, so we can edit this step to prevent confusion and tickets.